### PR TITLE
remove double wording

### PR DESCRIPTION
**Pull request by bartvermeersch**

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/configure-the-connection-filter-policy.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ description: "Admins can learn how to configure connection filtering in Exchange
...
+- **IP Allow List**: Skip spam filtering for all incoming messages from the source email servers that you specify by IP address or IP address range. For scenarios where spam filtering might still occur on messages from these sources, see the [Scenarios where messages from sources in the IP Allow List are still filtered](#scenarios-where-messages-from-sources-in-the-ip-allow-list-are-still-filtered) section later in this topic. For more information about how the IP Allow List should fit into your overall safe senders strategy, see [Create safe sender lists in EOP](create-safe-sender-lists-in-office-365.md).
 
 - **IP Block List**: Block all incoming messages from the source email servers that you specify by IP address or IP address range. The incoming messages are rejected, are not marked as spam, and no additional filtering occurs .For more information about how the IP Block List should fit into your overall blocked senders strategy, see [Create block sender lists in EOP](create-block-sender-lists-in-office-365.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-the-connection-filter-policy.md]:
```suggestion
- **IP Block List**: Block all incoming messages from the source email servers that you specify by IP address or IP address range. The incoming messages are rejected, are not marked as spam, and no additional filtering occurs. For more information about how the IP Block List should fit into your overall blocked senders strategy, see [Create block sender lists in EOP](create-block-sender-lists-in-office-365.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please merge additional edit.

**Comment by mypil** [comment]
@bartvermeersch  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@chrisda  - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**